$NetBSD: patch-ao,v 1.1 2013/02/02 05:06:43 makoto Exp $

Avoid following ERROR:
ERROR: [check-portability.awk] gdb/config/djgpp/djconfig.sh: if test "`pwd`" == "${srcdir}" ; then

--- gdb/config/djgpp/djconfig.sh.orig	2002-02-25 17:09:45.000000000 +0100
+++ gdb/config/djgpp/djconfig.sh	2006-10-22 09:54:26.000000000 +0200
@@ -76,7 +76,7 @@ TMPFILE="${TMPDIR-.}/cfg.tmp"
 
 # We need to skip the build directory if it is a subdirectory of $srcdir,
 # otherwise we will have an infinite recursion on our hands...
-if test "`pwd`" == "${srcdir}" ; then
+if test "`pwd`" = "${srcdir}" ; then
   SKIPDIR=""
   SKIPFILES=""
 else
